Dockerfile

```diff
@@ -0,0 +1,51 @@
+# ==============================
+FROM helsinkitest/python:3.7-slim as appbase
+# ==============================
+RUN mkdir /entrypoint
+
+COPY --chown=appuser:appuser requirements.txt /app/requirements.txt
+COPY --chown=appuser:appuser requirements-prod.txt /app/requirements-prod.txt
+
+RUN apt-install.sh \
+        git \
+        netcat \
+        libpq-dev \
+        build-essential \
+        gdal-bin \
+        python-gdal \
+        python3-gdal \
+    && pip install -U pip \
+    && pip install --no-cache-dir -r /app/requirements.txt \
+    && pip install --no-cache-dir  -r /app/requirements-prod.txt \
+    && apt-cleanup.sh build-essential
+
+COPY --chown=appuser:appuser docker-entrypoint.sh /entrypoint/docker-entrypoint.sh
+ENTRYPOINT ["/entrypoint/docker-entrypoint.sh"]
+
+# ==============================
+FROM appbase as development
+# ==============================
+
+COPY --chown=appuser:appuser requirements-dev.txt /app/requirements-dev.txt
+RUN apt-install.sh \
+        build-essential \
+    && pip install --no-cache-dir -r /app/requirements-dev.txt \
+    && apt-cleanup.sh build-essential
+
+ENV DEV_SERVER=1
+
+COPY --chown=appuser:appuser . /app/
+
+USER appuser
+EXPOSE 8083/tcp
```

Comment:
The port here does not match the port in `docker-compose.yml` but it doesn't matter since `EXPOSE` in a no-op anyways.

Reply:
Oops.